### PR TITLE
FC fix and update translation

### PR DESCRIPTION
**Pull request by Bambooin**
**You should replace the real qq group if you approve this PR, as I can't find the qq number.**

### CODE REVIEW

File: res/values-en/strings.xml [hunk 1]
```diff
@@ -38,15 +38,15 @@
     <string name="pref_credits">Credits</string>
     <string name="pref_pime">Win10 PIME Platform</string>
 
-    <string name="pref_input">Input</string>
+    <string name="pref_input">Input Setting</string>
     <string name="pref_enable">Enable</string>
     <string name="pref_enable_summary">Enable TRIME</string>
-    <string name="pref_select">Select</string>
+    <string name="pref_select">Change Keyboard</string>
```
Comment (osfans) [inline res/values-en/strings.xml:44]: 這個不是改鍵盤吧？是改輸入法。

File: res/values/strings.xml [hunk 2]
```diff
@@ -25,8 +25,10 @@
     <string name="pref_help">求助</string>
     <string name="pref_wiki">維基教程</string>
     <string name="pref_issues">問題反饋</string>
-    <string name="pref_qq1">Rime菜鳥QQ羣</string>
-    <string name="pref_qq2">中州韻QQ羣</string>
+    <string name="pref_rime_qq_newbie">Rime菜鳥QQ羣</string>
+    <string name="pref_rime_qq_newbie_summary" translatable="false">qq group number</string>
```
Comment (osfans) [inline res/values/strings.xml]: 523571568

File: res/values/strings.xml [hunk 3]
```diff
@@ -25,8 +25,10 @@
...
+    <string name="pref_rime_qq">中州韻QQ羣</string>
+    <string name="pref_rime_qq_summary" translatable="false">qq group number</string>
```
Comment (osfans) [inline res/values/strings.xml]: 77608640

File: res/values-en/strings.xml (hunk 1 above)
Comment (Bambooin) [reply res/values-en/strings.xml:44]: I refer the text in Android Setting, and you select it the pop view title is **Change Keyboard** if you setting your Android to use english.

Comment (osfans) [reply res/values-en/strings.xml:44]: 還真是，那就用它吧。

**Pull request merged**